api/admin/controller.py

```diff
@@ -1071,26 +1072,26 @@ def admin_auth_services(self):
 
             return dict(
                 admin_auth_services=auth_services,
-                providers=ExternalIntegration.ADMIN_AUTH_PROVIDERS,
+                providers=ExternalIntegration.ADMIN_AUTH_PROTOCOLS,
             )
 
-        provider = flask.request.form.get("provider")
-        if not provider:
+        protocol = flask.request.form.get("provider")
```

Comment:
That's fine. This doesn't affect what's displayed to users, but if we did want to change it we'd also have to update the form in circulation-web.